programs/drift/src/controller/orders.rs

```diff
@@ -354,6 +332,38 @@ pub fn place_perp_order(
     Ok(())
 }
 
+fn get_auction_prices(
+    params: &OrderParams,
+    oracle_price_data: &OraclePriceData,
+    tick_size: u64,
+) -> DriftResult<(i64, i64)> {
+    let (auction_start_price, auction_end_price) =
+        if matches!(params.order_type, OrderType::Market | OrderType::Oracle) {
```

Comment:
early return with `if !matches!(params.order_type, OrderType::Market | OrderType::Oracle) {` would probs be cleaner imo - would save a `let (auction_start_price, auction_end_price) =`

Reply:
yeah i like that